data/xapleaux/xapleaux missions.txt

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2022 by MasterOfGrey
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Xapleaux: First Contact"
+	landing
+	source
+		government "Xapleaux"
+		not attributes "uninhabited"
+	on offer
+		conversation
+			`The brightly illuminated world rolls beneath you as your ship approaches the spaceport.`
+			branch revisit
+				has "Met X Already"
+			`	Disembarking your ship to meet these strange new aliens, you are greeted by the unnerving sight of what appears to be foliage moving and shifting in a wide circle around you. It is only as a few of them move forward to approach you that you realize the aliens are the foliage, or at least the foliage is growing on them. It's rather hard to tell.`
+			`	Three of them approach you, and it becomes clear that they have been keeping their distance so far as your eyes realize that these aliens are most emphatically not small. Their front-most portion rises tall enough to make most professional basketball players have to look up slightly, and that is before considering the leaf-like fronds, mushroom like bulges, or other less identifiable structures which appear to be growing up from their bodies, in some cases nearly a third as high again from the top of their heads.`

```

Comment:
```suggestion
			`	Three of them approach you, and it becomes clear that they have been keeping their distance so far as you realize that these aliens are most emphatically not small. Their front-most portion rises tall enough to make most professional basketball players have to look up slightly, and that is before considering the leaf-like fronds, mushroom like bulges, or other less identifiable structures which appear to be growing up from their bodies, in some cases nearly a third as high again from the top of their heads.`
```
```suggestion
			`	Three of them approach you, and it becomes clear that they have been keeping their distance so far as your eyes inform you that these aliens are most emphatically not small. Their front-most portion rises tall enough to make most professional basketball players have to look up slightly, and that is before considering the leaf-like fronds, mushroom like bulges, or other less identifiable structures which appear to be growing up from their bodies, in some cases nearly a third as high again from the top of their heads.`
```
"As your eyes realize" doesn't make sense - to realise is to become aware of or to understand, and that happens in the brain, not the eyes.